site/frontend/src/components/cachegrind-cmd.vue

```diff
@@ -0,0 +1,61 @@
+<script setup lang="ts">
+import {CompileTestCase, Profile} from "../pages/compare/compile/common";
+import {computed} from "vue";
+
+const props = defineProps<{
+  commit: string;
+  testCase: CompileTestCase;
+  baseline_commit?: string;
+}>();
+
+const firstCommit = computed(() => {
+  if (props.baseline_commit !== undefined) {
+    return props.baseline_commit;
+  } else {
+    return props.commit;
+  }
+});
+
+function normalizeProfile(profile: Profile): string {
+  if (profile === "opt") {
+    return "Opt";
+  } else if (profile === "debug") {
+    return "Debug";
+  } else if (profile === "check") {
+    return "Check";
+  } else if (profile === "doc") {
+    return "Doc";
+  }
+  return "<invalid profile>";
+}
+function normalizeScenario(scenario: string): string {
+  if (scenario === "full") {
+    return "Full";
+  } else if (scenario === "incr-full") {
+    return "IncrFull";
+  } else if (scenario === "incr-unchanged") {
+    return "IncrUnchanged";
+  } else if (scenario.startsWith("incr-patched")) {
+    return "IncrPatched";
+  }
+  return "<invalid scenario>";
+}
+</script>
+
+<template>
+  <pre><code>cargo run --bin collector profile_local cachegrind \
```

Comment:
I assume it's not slower or impactful to run collection in debug just to launch a few cargo and valgrind commands, rather than release? We tell people to run the collector in --release today.

Reply:
I usually just everything in debug locally, I don't think that it should affect a lot of things. But of course, running in release is theoretically *safer*. I can add `--release` here if you want.

Reply:
Yes let's, just because otherwise the detailed query page and this command would be different. We can figure out which of the two is best in the future if need be, but until then let's keep it consistent.